internal/executor/cache.go

```diff
@@ -90,16 +131,21 @@ func DownloadCache(executor *Executor, commandName string, cacheHost string, ins
 		Cache{
 			Name:           commandName,
 			Key:            cacheKey,
-			Folder:         folderToCache,
-			FolderHash:     folderToCacheHash,
-			FilesHashes:    folderToCacheFileHashes,
+			BaseFolder:     baseFolder,
+			FoldersToCache: foldersToCache,
+			Glob:           glob,
+			FileHasher:     fileHasher,
 			SkipUpload:     cacheAvailable && !instruction.ReuploadOnChanges,
 			CacheAvailable: cacheAvailable,
 		},
 	)
 	return true
 }
 
+func pathLooksLikeGlob(path string) bool {
+	return strings.Contains(path, "**")
```

Comment:
Do you think just checking for a single `*` will be enough?

Reply:
It probably will, see e6114c0.